sretoolbox/container/image.py

```diff
@@ -353,9 +382,17 @@ def _request_get(self, url):
 
             # Try again, this time with the Authorization header
             headers['Authorization'] = self._get_auth(www_auth)
-            response = requests.get(url, headers=headers)
+            response = requests.head(url, headers=headers)
+
+        if 'etag' in response.headers:
+            with contextlib.suppress(KeyError):
```

Comment:
this will likelly change with the previous suggestion, but I'm not convinced we need to import `contextlib` just to have a shorter try/except block.

Reply:
See my response to that suggestion.

Reply:
How is the `contextlib.suppress` better than a try/except block? Is that just for readability?

Reply:
Yes, it's a readability improvement.